src/pages/waypoint/commands/[[...page]].tsx

```diff
@@ -1,7 +1,8 @@
 import { generateStaticPaths, generateStaticProps } from 'layouts/docs/server'
 import waypointConfig from '../../../../config/waypoint.json'
 import DocsLayout from 'layouts/docs'
-import DocsPage from 'components/docs-page'
+import { MDXRemote } from 'next-mdx-remote'
```

Comment:
As far as I could tell, `DocsPage` only role was setting up Waypoint-specific MDX components, so for now have replaced with MDXRemote directly (and removed `components/docs-page`). Definitely expecting to refactor this during work to reduce complexity in MDX custom component config.